test/unit/org/apache/cassandra/db/compaction/UnifiedCompactionStrategyTest.java

```diff
@@ -1468,8 +1585,8 @@ public void testBucketSelectionMissing()
     @Test
     public void testBucketSelectionHalvesMissing()
     {
-        // Drop one half: still compact because of overlap
-        testBucketSelection(repeats(4, arr(6, 3)), arr(5, 6, 6), Overlaps.InclusionMethod.TRANSITIVE, 0, 1);
+        // Drop one half: don't compact
```

Comment:
re-ordering overlaps based on size would fail the test and now it doesn't compact..

Reply:
This comment change should be reverted.